connector/client/parse_query.go

```diff
@@ -86,74 +86,90 @@ func ParseQueryString(queryString string) (panels []*models.Panel, err error) {
 				}
 
 				for i := 0; i < int(multiplier); i++ {
-					newPanel.Items = append(newPanel.Items, items...)
+					newPanel.ConceptItems = append(newPanel.ConceptItems, conceptItems...)
+					newPanel.PatientSetIDItems = append(newPanel.PatientSetIDItems, psIDItems...)
 				}
 
 			} else {
 
-				items, err := ParseQueryItem(queryItem)
+				conceptItems, psIDItems, err := ParseQueryItem(queryItem)
 				if err != nil {
 					return nil, err
 				}
 
-				newPanel.Items = append(newPanel.Items, items...)
+				newPanel.ConceptItems = append(newPanel.ConceptItems, conceptItems...)
+				newPanel.PatientSetIDItems = append(newPanel.PatientSetIDItems, psIDItems...)
 			}
 		}
 		panels = append(panels, &newPanel)
 	}
 	return
 }
 
-// ParseQueryItem parses a string into an item
+// ParseQueryItem parses a string into an item.
```

Comment:
The [MedCo Documentation](https://ldsec.gitbook.io/medco-documentation/system-administrators/cli) should be updated accordingly, either when merged, or a v2.0.2 branch can already be created.